providers/openebs/installers/operator/0.6/ansible/openebs_setup.yaml

```diff
@@ -44,6 +42,34 @@
               replace: "image: {{ lookup('env','OPENEBS_SNAPSHOT_PROVISIONER_IMAGE') }}"
             when: lookup('env','OPENEBS_SNAPSHOT_PROVISIONER_IMAGE') | length > 0
 
+          - name: Updating openebs jiva controller image
+            replace:
+              path: "openebs-operator.yaml"
+              regexp: "        - name: OPENEBS_IO_JIVA_CONTROLLER_IMAGE\n          value: \"openebs/jiva.*"
+              replace: "        - name: OPENEBS_IO_JIVA_CONTROLLER_IMAGE\n          value: \"{{ lookup('env','OPENEBS_IO_JIVA_CONTROLLER_IMAGE') }}\""
+            when: lookup('env','OPENEBS_IO_JIVA_CONTROLLER_IMAGE') | length > 0
+
+          - name: Updating openebs jiva replica image
+            replace:
+              path: "openebs-operator.yaml"
+              regexp: "        - name: OPENEBS_IO_JIVA_REPLICA_IMAGE\n          value: \"openebs/jiva.*"
+              replace: "        - name: OPENEBS_IO_JIVA_REPLICA_IMAGE\n          value: \"{{ lookup('env','OPENEBS_IO_JIVA_REPLICA_IMAGE') }}\""
+            when: lookup('env','OPENEBS_IO_JIVA_REPLICA_IMAGE') | length > 0
+
+          - name: Updating openebs io volume monitor image
+            replace:
+              path: "openebs-operator.yaml"
+              regexp: "        - name: OPENEBS_IO_VOLUME_MONITOR_IMAGE\n          value: \"openebs/m-exporter.*"
+              replace: "        - name: OPENEBS_IO_VOLUME_MONITOR_IMAGE\n          value: \"{{ lookup('env','OPENEBS_IO_VOLUME_MONITOR_IMAGE') }}\""
+            when: lookup('env','OPENEBS_IO_VOLUME_MONITOR_IMAGE') | length > 0
+
+          - name: Updating jiva replica count
+            replace:
+              path: "openebs-operator.yaml"
+              regexp: "        - name: OPENEBS_IO_JIVA_REPLICA_COUNT\n          value: \"3\""
+              replace: "        - name: OPENEBS_IO_JIVA_REPLICA_COUNT\n          value: \"{{ lookup('env','OPENEBS_IO_JIVA_REPLICA_COUNT') }}\""
+            when: lookup('env','OPENEBS_IO_JIVA_REPLICA_COUNT') | length > 0
+

```

Comment:
Unlike the lineinfile module, the regex replace should not need the indents specified. However, I stand to be corrected. Can you confirm this ?

Reply:
I have not tried without space but i have checked its wotking in mycase with spaces

Reply:
OK